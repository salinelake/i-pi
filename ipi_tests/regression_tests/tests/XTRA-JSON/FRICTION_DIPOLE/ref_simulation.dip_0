 #*EXTRAS*# Step:           0  Bead:      0  
     0.00000000           0.00000000           0.00000000
 #*EXTRAS*# Step:           1  Bead:      0  
    -0.00000035          -0.00000000          -0.00000000
 #*EXTRAS*# Step:           2  Bead:      0  
    -0.00000280          -0.00000001          -0.00000000
 #*EXTRAS*# Step:           3  Bead:      0  
    -0.00000943          -0.00000002          -0.00000000
 #*EXTRAS*# Step:           4  Bead:      0  
    -0.00002224          -0.00000005          -0.00000001
 #*EXTRAS*# Step:           5  Bead:      0  
    -0.00004316          -0.00000010          -0.00000001
 #*EXTRAS*# Step:           6  Bead:      0  
    -0.00007402          -0.00000018          -0.00000002
 #*EXTRAS*# Step:           7  Bead:      0  
    -0.00011649          -0.00000029          -0.00000003
 #*EXTRAS*# Step:           8  Bead:      0  
    -0.00017209          -0.00000043          -0.00000005
 #*EXTRAS*# Step:           9  Bead:      0  
    -0.00024215          -0.00000061          -0.00000006
