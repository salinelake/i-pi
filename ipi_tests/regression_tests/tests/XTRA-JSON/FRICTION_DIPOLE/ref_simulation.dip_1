 #*EXTRAS*# Step:           0  Bead:      1  
     0.00000000           0.00000000           0.00000000
 #*EXTRAS*# Step:           1  Bead:      1  
    -0.00000003          -0.00000105           0.00000001
 #*EXTRAS*# Step:           2  Bead:      1  
    -0.00000027          -0.00000834           0.00000007
 #*EXTRAS*# Step:           3  Bead:      1  
    -0.00000091          -0.00002769           0.00000023
 #*EXTRAS*# Step:           4  Bead:      1  
    -0.00000217          -0.00006419           0.00000054
 #*EXTRAS*# Step:           5  Bead:      1  
    -0.00000429          -0.00012179           0.00000101
 #*EXTRAS*# Step:           6  Bead:      1  
    -0.00000752          -0.00020311           0.00000166
 #*EXTRAS*# Step:           7  Bead:      1  
    -0.00001216          -0.00030921           0.00000250
 #*EXTRAS*# Step:           8  Bead:      1  
    -0.00001853          -0.00043949           0.00000349
 #*EXTRAS*# Step:           9  Bead:      1  
    -0.00002700          -0.00059174           0.00000460
