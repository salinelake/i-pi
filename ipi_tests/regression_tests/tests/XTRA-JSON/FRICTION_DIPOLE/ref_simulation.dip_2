 #*EXTRAS*# Step:           0  Bead:      2  
     0.00000000           0.00000000           0.00000000
 #*EXTRAS*# Step:           1  Bead:      2  
    -0.00000355           0.00000004          -0.00000051
 #*EXTRAS*# Step:           2  Bead:      2  
    -0.00002838           0.00000031          -0.00000407
 #*EXTRAS*# Step:           3  Bead:      2  
    -0.00009559           0.00000104          -0.00001351
 #*EXTRAS*# Step:           4  Bead:      2  
    -0.00022599           0.00000241          -0.00003133
 #*EXTRAS*# Step:           5  Bead:      2  
    -0.00043991           0.00000457          -0.00005946
 #*EXTRAS*# Step:           6  Bead:      2  
    -0.00075707           0.00000762          -0.00009920
 #*EXTRAS*# Step:           7  Bead:      2  
    -0.00119642           0.00001159          -0.00015109
 #*EXTRAS*# Step:           8  Bead:      2  
    -0.00177599           0.00001646          -0.00021488
 #*EXTRAS*# Step:           9  Bead:      2  
    -0.00251282           0.00002213          -0.00028952
