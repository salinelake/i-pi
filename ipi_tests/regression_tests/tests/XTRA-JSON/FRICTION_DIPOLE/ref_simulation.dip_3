 #*EXTRAS*# Step:           0  Bead:      3  
     0.00000000           0.00000000           0.00000000
 #*EXTRAS*# Step:           1  Bead:      3  
    -0.00000009           0.00000061          -0.00000000
 #*EXTRAS*# Step:           2  Bead:      3  
    -0.00000069           0.00000482          -0.00000001
 #*EXTRAS*# Step:           3  Bead:      3  
    -0.00000235           0.00001600          -0.00000005
 #*EXTRAS*# Step:           4  Bead:      3  
    -0.00000560           0.00003706          -0.00000011
 #*EXTRAS*# Step:           5  Bead:      3  
    -0.00001101           0.00007029          -0.00000021
 #*EXTRAS*# Step:           6  Bead:      3  
    -0.00001919           0.00011716          -0.00000035
 #*EXTRAS*# Step:           7  Bead:      3  
    -0.00003077           0.00017823          -0.00000055
 #*EXTRAS*# Step:           8  Bead:      3  
    -0.00004644           0.00025312          -0.00000081
 #*EXTRAS*# Step:           9  Bead:      3  
    -0.00006695           0.00034047          -0.00000114
