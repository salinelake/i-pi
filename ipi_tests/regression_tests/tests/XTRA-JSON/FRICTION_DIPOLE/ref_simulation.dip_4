 #*EXTRAS*# Step:           0  Bead:      4  
     0.00000000           0.00000000           0.00000000
 #*EXTRAS*# Step:           1  Bead:      4  
    -0.00000013          -0.00000035          -0.00000000
 #*EXTRAS*# Step:           2  Bead:      4  
    -0.00000103          -0.00000279          -0.00000001
 #*EXTRAS*# Step:           3  Bead:      4  
    -0.00000347          -0.00000927          -0.00000003
 #*EXTRAS*# Step:           4  Bead:      4  
    -0.00000819          -0.00002146          -0.00000006
 #*EXTRAS*# Step:           5  Bead:      4  
    -0.00001589          -0.00004062          -0.00000012
 #*EXTRAS*# Step:           6  Bead:      4  
    -0.00002725          -0.00006757          -0.00000020
 #*EXTRAS*# Step:           7  Bead:      4  
    -0.00004288          -0.00010254          -0.00000031
 #*EXTRAS*# Step:           8  Bead:      4  
    -0.00006334          -0.00014522          -0.00000044
 #*EXTRAS*# Step:           9  Bead:      4  
    -0.00008912          -0.00019470          -0.00000060
