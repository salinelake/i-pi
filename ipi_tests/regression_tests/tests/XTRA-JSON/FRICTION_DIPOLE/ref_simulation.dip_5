 #*EXTRAS*# Step:           0  Bead:      5  
     0.00000000           0.00000000           0.00000000
 #*EXTRAS*# Step:           1  Bead:      5  
     0.00000048          -0.00000000          -0.00000000
 #*EXTRAS*# Step:           2  Bead:      5  
     0.00000385          -0.00000001          -0.00000000
 #*EXTRAS*# Step:           3  Bead:      5  
     0.00001295          -0.00000002          -0.00000001
 #*EXTRAS*# Step:           4  Bead:      5  
     0.00003049          -0.00000004          -0.00000001
 #*EXTRAS*# Step:           5  Bead:      5  
     0.00005904          -0.00000008          -0.00000003
 #*EXTRAS*# Step:           6  Bead:      5  
     0.00010097          -0.00000014          -0.00000005
 #*EXTRAS*# Step:           7  Bead:      5  
     0.00015838          -0.00000022          -0.00000007
 #*EXTRAS*# Step:           8  Bead:      5  
     0.00023308          -0.00000032          -0.00000010
 #*EXTRAS*# Step:           9  Bead:      5  
     0.00032655          -0.00000045          -0.00000014
