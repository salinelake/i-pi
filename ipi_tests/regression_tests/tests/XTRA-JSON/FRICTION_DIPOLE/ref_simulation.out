# column   1     --> step : The current simulation time step.
# column   2     --> conserved : The value of the conserved energy quantity per bead.
    0.00000000e+00     6.35780316e-03   
    1.00000000e+00     6.35782751e-03   
    2.00000000e+00     6.35789994e-03   
    3.00000000e+00     6.35801855e-03   
    4.00000000e+00     6.35818025e-03   
    5.00000000e+00     6.35838083e-03   
    6.00000000e+00     6.35861506e-03   
    7.00000000e+00     6.35887685e-03   
    8.00000000e+00     6.35915938e-03   
    9.00000000e+00     6.35945529e-03   
